### ISSUE TITLE
How to change/check config

### ISSUE THREAD

**Issue by jirisykora83** [TimSoethout/goodwe-sems-home-assistant#105]
How to change/check config?

It seems that currently it is not possible edit config already added entities (at least via UI).

**Comment by TimSoethout** [comment]
Correct, would be nice feature request. Let's see what copilot makes of it.
